component-library/src/scss/_documentation-pages.scss

```diff
@@ -0,0 +1,242 @@
+@use '../../../core-library/util/device' as device;
+
+.content-doc-wrapper {
+  box-sizing: border-box;
+  max-width: 900px;
+  margin: 0 auto;
+
+  img {
+    width: 100%;
+  }
+}
+
+h3 {
+  display: inline-block;
+  font-weight: 700;
+
+  &.primary {
+    font-size: 36px;
+  }
+
+  &.secondary {
+    font-size: 28px;
+    margin-top: 60px;
+  }
+}
+
+.body1.intro {
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+.body3.demo {
+  margin-top: 0px;
+  margin-bottom: 24px;
+}
+
+.subtitles {
+  margin-bottom: 12px;
+}
+
+.body3.subtext-1 {
+  margin-top: 12px;
+  margin-bottom: 24px;
+}
+
+.body3.subtext-2 {
+  margin-top: 12px;
+  margin-bottom: 32px;
+}
+
+.body3.subtext-3 {
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.body3.subtext-4 {
+  margin-top: 8px;
+  margin-bottom: 32px;
+}
+
+.body3.subtext-5 {
+  margin-top: 8px;
+  margin-bottom: 24px;
+}
+
+.body3.subtext-6 {
+  margin-top: 8px;
+  margin-bottom: 48px;
+}
+
+.body3.subtext-7 {
+  margin-top: 8px;
+  margin-bottom: 44px;
+}
+
+.body3.subtext-8 {
+  margin-top: 8px;
+  margin-bottom: 0px;
+}
+
+.body3.subtext-9 {
+  margin-top: 8px;
+  margin-bottom: 12px;
+}
+
+.body3.subtext-10 {
+  margin-top: 8px;
+  margin-bottom: 68px;
+}
+
+.body3.subtext-11 {
+  margin-top: 8px;
+  margin-bottom: 88px;
+}
+
+.body3.subtext-12 {
+  margin-top: 8px;
+  margin-bottom: 52px;
+}
+
+.body3.subtext-13 {
+  margin-top: 8px;
+  margin-bottom: 0px;
+}
+
+.body3.last {
+  margin-top: 12px;
+  margin-bottom: 0px;
+}
+
+.body3.anatomy {
+  margin-bottom: 0px;
+}
+
+.config-heading {
+  margin-top: 24px;
+  margin-bottom: 12px;
+}
+
+ul {
+  margin-top: 0px;
+  margin-bottom: 12px;
+}
+
+.end-list {
+  margin-bottom: 0px;
+}
+
+> a {
+  margin-left: 16px;
+  position: relative;
+  bottom: 5px;
+  text-decoration: none;
+  color: var(--link-text);
+  cursor: pointer;
+}
+
+h5 {
+  font-weight: 700;
+  margin-bottom: 12px;
+  margin-top: 0px;
+
+  &.text-underline {
+    border-bottom: 1px solid;
+    margin-top: 8px;
+    margin-bottom: 0px;
+  }
+
+  &.text-color-green {
+    color: var(--success-text);
+  }
+
+  &.text-color-red {
+    color: var(--critical-text);
+  }
+
+  &.round-purple-num {
+    display: inline-block;
+    margin-left: 8px;
+    margin-top: 24px;
+    margin-bottom: 8px;
+    position: relative;
+    top: 2px;
+  }
+}
+
+span {
+  &.round-purple-num {
```

Comment:
I missed this in my story - can you please replace the hex colours for the purple circles as follows:

`span {
  &.round-purple-num {
    color: var(--purple-9);
    font-weight: 700;
    background-color: var(--purple-2);
    padding: 9px 15px;
    border-radius: 50%;
  }
}`

Reply:
thank you. fixed.